ingestion/src/kvkafka.rs

```diff
@@ -94,6 +110,9 @@ fn create_consumer(brokers: &str, group_id: &str, topic: &str) -> LoggingConsume
         .set("auto.commit.interval.ms", "5000")
         // but only commit the offsets explicitly stored via `consumer.store_offset`.
         .set("enable.auto.offset.store", "false")
+        // if we don't have a starting offset, or it's out of range, start from the earliest
+        // available on the cluster
+        .set("auto.offset.reset", "earliest")
```

Comment:
Is there a way to explicitly set the offset? We had this for reingesting before, but it tried to "eat both ends" at once to get the new data while catching up with the old data...

Reply:
The broker stores the offset based on your consumer group, which we set as "lard" in the systemd unit. This option only applies if the broker doesn't have a saved offset for that group or it's not in range. If we want to reingest, we change the consumer group name, which is the same as what we did in ODA

Reply:
Note: the default for this option is "latest" (well actually "largest", which is a synonym for "latest"), which will start reading from the front of the queue, which we definitely don't want.

Documentation for these config options can be found here:
https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md